app/main/views/briefs.py

```diff
@@ -0,0 +1,162 @@
+    validate_brief_data(brief, enforce_required=False, required_fields=page_questions)
+
+    db.session.add(brief)
+    try:
+        db.session.flush()
+    except IntegrityError as e:
+        db.session.rollback()
+        abort(400, e.orig)
+
+    audit = AuditEvent(
+        audit_type=AuditTypes.create_brief,
+        user=updater_json['updated_by'],
+        data={
+            'briefId': brief.id,
+            'briefJson': brief_json,
+        },
+        db_object=brief,
+    )
+
+    db.session.add(audit)
+
+    db.session.commit()
+
+    return jsonify(briefs=brief.serialize()), 201
+
+
+@main.route('/briefs/<int:brief_id>', methods=['POST'])
+def update_brief(brief_id):
+    updater_json = validate_and_return_updater_request()
+    page_questions = get_request_page_questions()
+
+    json_payload = get_json_from_request()
+    json_has_required_keys(json_payload, ['briefs'])
+    brief_json = json_payload['briefs']
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    if brief.status == 'live':
+        abort(400, "Cannot update a live brief")
+
+    brief.update_from_json(brief_json)
+
+    validate_brief_data(brief, enforce_required=False, required_fields=page_questions)
+
+    audit = AuditEvent(
+        audit_type=AuditTypes.update_brief,
+        user=updater_json['updated_by'],
+        data={
+            'briefId': brief.id,
+            'briefJson': brief_json,
+        },
+        db_object=brief,
+    )
+
+    db.session.add(brief)
+    db.session.add(audit)
+    db.session.commit()
+
+    return jsonify(briefs=brief.serialize()), 200
+
+
+@main.route('/briefs/<int:brief_id>', methods=['GET'])
+def get_brief(brief_id):
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    return jsonify(briefs=brief.serialize())
+
+
+@main.route('/briefs', methods=['GET'])
+def list_briefs():
+    page = get_valid_page_or_1()
+
+    briefs = Brief.query.order_by(Brief.id)
+    briefs = briefs.paginate(
+        page=page,
+        per_page=current_app.config['DM_API_BRIEFS_PAGE_SIZE'])
+
+    return jsonify(
+        briefs=[brief.serialize() for brief in briefs.items],
+        links=pagination_links(
+            briefs,
+            '.list_briefs',
+            request.args
+        )
+    )
+
+
+@main.route('/briefs/<int:brief_id>/status', methods=['PUT'])
+def update_brief_status(brief_id):
+    updater_json = validate_and_return_updater_request()
+
+    json_payload = get_json_from_request()
+    json_has_required_keys(json_payload, ['briefs'])
+    brief_json = json_payload['briefs']
+    json_has_required_keys(brief_json, ['status'])
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    if brief_json['status'] == brief.status:
+        abort(400, "Brief is already {}".format(brief.status))
+
+    brief.status = brief_json['status']
+
+    validate_brief_data(brief, enforce_required=True)
```

Comment:
We only need to do this when making the brief `live`, although I suppose it's the only status change we could have right now.

One other thing we could check here is that the framework is still 'live', so that it's not possible to create a draft brief and then publish it once the framework has expired.